<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-M6K6JG6B" failures="1" tests="3" name="Test" time="23.471" errors="0" timestamp="2022-07-09T18:42:43 IST">
  <testcase classname="testCases.NegativeTestCase" name="stringInput" time="3.275"/>
  <testcase classname="testCases.EdgeTestCase" name="highestIntegerNumber" time="3.259">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at base.BaseClass.checkOutput(BaseClass.java:40)
at base.BaseClass.calculateFactorial(BaseClass.java:46)
at testCases.EdgeTestCase.highestIntegerNumber(EdgeTestCase.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- highestIntegerNumber -->
  <testcase classname="testCases.PositiveTestCase" name="postiveNumber" time="3.251"/>
</testsuite> <!-- Test -->
